src/Service.Tests/Configuration/RuntimeConfigLoaderTests.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.IO.Abstractions.TestingHelpers;
+using System.Threading.Tasks;
+using Azure.DataApiBuilder.Config;
+using Azure.DataApiBuilder.Config.ObjectModel;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Azure.DataApiBuilder.Service.Tests.Configuration;
+
+[TestClass]
+public class RuntimeConfigLoaderTests
+{
+    [DataTestMethod]
+    [DataRow("dab-config.CosmosDb_NoSql.json")]
+    [DataRow("dab-config.MsSql.json")]
+    [DataRow("dab-config.MySql.json")]
+    [DataRow("dab-config.PostgreSql.json")]
+    public async Task CanLoadStandardConfig(string configPath)
```

Comment:
is the test just validating that `RuntimeConfigLoader` can process the file from the filesystem? Is it also confirming all the jsonconverters worke?